lib/src/broadcast/user_interaction_bloc.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:bloc/bloc.dart';
+
+enum UserInteractionEvent{ none, mute_toggle, }
+
+class UserInteractionCubit extends Cubit<UserInteractionEvent> {
+  UserInteractionCubit() : super(UserInteractionEvent.none);
+
+  toggle() => emit(UserInteractionEvent.mute_toggle);
```

Comment:
Just an idea, but when you're working with events it might be better to just use a bloc instead of a cubit?